tracer/src/Datadog.Trace/Ci/TestSession.cs

```diff
@@ -38,7 +39,7 @@ private TestSession(string? command, string? workingDirectory, string? framework
         var environment = CIEnvironmentValues.Instance;
 
         Command = command;
-        WorkingDirectory = workingDirectory ?? Environment.CurrentDirectory;
+        WorkingDirectory = workingDirectory ?? System.Environment.CurrentDirectory;
```

Comment:
nit: I feel like this is going to get old for you fast 😅 I would be inclined to change the CI namespace to something like `Datadog.Trace.Ci.CiEnvironment`; it feels like you're just causing pain for yourself with little benefit currently!

Reply:
you're right, done that in [8b8b3f1](https://github.com/DataDog/dd-trace-dotnet/pull/5226/commits/8b8b3f1e3a2a544ae51a0ae600d9e1c13f70f1d6)